root/var/lib/asterisk/bin/googletts_sendmail.php

```diff
@@ -0,0 +1,201 @@
+stream.part1 - header after the bounday
+stream.part2 - body of the message
+stream.part3 - attachment in base64 (WAV file)
+stream.part4 - footer of the message
+*/
+$shellCmd = "awk '/'" . $boundary . "'/{i++}{print > \"" . $tempDir . "/stream.part\"i}' " . $tempStdinFile;
+$awkReturn = trim(shell_exec($shellCmd));
+
+# if mail is having no audio attachment (plain text)
+$shellCmd = "cat " . $tempDir . "/stream.part1 | grep 'plain'";
+$plainText = trim(shell_exec($shellCmd));
+
+// with audio file
+if ($plainText == "") {
+	# cut the attachment into parts
+	# stream.part3.head - header of attachment
+	$shellCmd = "sed '7,\$d' " . $tempDir . "/stream.part3 > " . $tempDir . "/stream.part3.wav.head";
+	$cmdReturn = trim(shell_exec($shellCmd));
+
+	# stream.part3.wav.base64 - wav file of attachment (encoded base64)
+	$shellCmd = "sed '1,6d' " . $tempDir . "/stream.part3 > " . $tempDir . "/stream.part3.wav.base64";
+	$cmdReturn = trim(shell_exec($shellCmd));
+
+	# convert the base64 file to a wav file
+	$shellCmd = "dos2unix -o " . $tempDir . "/stream.part3.wav.base64";
+	$cmdReturn = trim(shell_exec($shellCmd));
+	//
+	$shellCmd = "base64 -di " . $tempDir . "/stream.part3.wav.base64 > " . $tempDir . "/stream.part3.wav";
+	$cmdReturn = trim(shell_exec($shellCmd));
+
+	# convert wave file (GSM encoded or not) to PCM wav file
+	$shellCmd = "sox " . $tempDir . "/stream.part3.wav " . $tempDir . "/stream.part3-pcm.wav";
+	$cmdReturn = trim(shell_exec($shellCmd));
+	/*
+	convert PCM wav file to mp3 file
+	-b 24 is using CBR, giving better compatibility on smartphones (you can use -b 32 to increase quality)
+	-V 2 is using VBR, a good compromise between quality and size for voice audio files
+	*/
+	$shellCmd = "lame -m m -b 24 " . $tempDir . "/stream.part3-pcm.wav " . $tempDir . "/stream.part3.mp3";
+	$cmdReturn = trim(shell_exec($shellCmd));
+
+	# convert back mp3 to base64 file
+	$shellCmd = "base64 " . $tempDir . "/stream.part3.mp3 > " . $tempDir . "/stream.part3.mp3.base64";
+	$cmdReturn = trim(shell_exec($shellCmd));
+	/*
+	generate the new mp3 attachment header
+	change Type: audio/x-wav or audio/x-WAV to Type: audio/mpeg
+	change name="msg----.wav" or name="msg----.WAV" to name="msg----.mp3"
+	*/
+	$shellCmd = "sed 's/x-[wW][aA][vV]/mpeg/g' " . $tempDir . "/stream.part3.wav.head | sed 's/.[wW][aA][vV]/.mp3/g' > " . $tempDir . "/stream.part3.mp3.head";
+	$cmdReturn = trim(shell_exec($shellCmd));
+
+	# convert wav file to flac compatible format for Google speech recognition
+	$shellCmd = "sox " . $tempDir . "/stream.part3.wav -r 16000 -b 16 -c 1 " . $tempDir . "/audio.flac lowpass -2 2500";
+	$cmdReturn = trim(shell_exec($shellCmd));
+
+	# [START speech_transcribe_sync]
+	$transcript = "";
+	$confidence = "";
+    $vm = \FreePBX::Voicemail()->getVoicemail(false);
+
+    # only runs speech recognition if the mailcmd is it
+    if (!empty($vm['general']['mailcmd']) && $vm['general']['mailcmd'] == '/var/lib/asterisk/bin/googletts_sendmail.php') {
```

Comment:
```suggestion
    if (!empty($vm['general']['mailcmd']) && $vm['general']['mailcmd'] == '/var/lib/asterisk/bin/googlestt_sendmail.php') {
```
stt, speech to text, not tts, text to speech